tests/wpt/metadata/fetch/api/cors/cors-preflight.any.js.ini

```diff
@@ -6,27 +6,6 @@
   [CORS [PUT\] [several headers\], server allows]
```

Comment:
These tests fail because of https://github.com/servo/servo/blob/a4bf7e51793813b2f5120c621a02771ad495be92/components/script/dom/headers.rs#L88-L100

Should i remove the leading space?

I'm not sure to understand the point `HyperHeaders might not expose the information we need to do it right`...

Reply:
I feel like there's possibly some history here. File an issue about this, please!

Reply:
Done #27765